packages/donor/src/screens/unsubscribe/UnsubscribedScreen.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import UnsubscribedScreen from "./UnsubscribedScreen";
+import { MemoryRouter } from "react-router-dom";
+import { Story } from "@storybook/react";
+import { UnsubscribedScreenProps } from "./UnsubscribedScreen";
+import { mockUser } from "../../utils/StorybookMocks";
+
+export default {
+  component: UnsubscribedScreen,
+  title: "Screens/Unsubscribed",
+  parameters: { layout: "fullscreen" },
+  decorators: [
+    (Story: any) => (
+      <MemoryRouter>
```

Comment:
I added MemoryRouter to `preview.tsx` so it is not needed here. I think you can remove the entire `decorators`